process.py

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/env python
```

Comment:
As discussed, remove this file from here and move it to git dir and rename it to GithubCommitsProcessor.